tests/macros/utils.js

```diff
@@ -176,17 +176,22 @@ function lintHTML(html, fragment = true) {
                 </html>`;
     }
     try {
-        execSync(`java -jar ${vnu} --errors-only --format text -`, {
+        execSync(`java -jar ${JSON.stringify(vnu)} --errors-only --format text -`, {
```

Comment:
Not&nbsp;using `JSON.stringify(…)` here results in&nbsp;a&nbsp;problem similar to&nbsp;https://github.com/mdn/kumascript/pull/1057#discussion_r256427946, but&nbsp;with spaces instead.

Reply:
That's a very useful comment, in https://github.com/mdn/kumascript/pull/1057#discussion_r256427946
Can you transpose that into a code comment right above line 179?